examples/cmdline/duk_cmdline.c

```diff
@@ -239,8 +239,9 @@ static duk_ret_t wrapped_compile_execute(duk_context *ctx, void *udata) {
 
 	if (src_data != NULL && src_len >= 2 && src_data[0] == (char) 0xff) {
 		/* Bytecode. */
-		duk_push_lstring(ctx, src_data, src_len);
-		duk_to_buffer(ctx, -1, NULL);
+		void *buf;
+		buf = duk_push_fixed_buffer(ctx, src_len);
+		memcpy(buf, (const void *) src_data, src_len);
```

Comment:
Yep, there's an internal use for such a call (though there weren't enough call sites to justify it based on footprint alone). Also such a call avoids unnecessary zeroing.